Documentation/Settings.md

```diff
@@ -0,0 +1,302 @@
+<!-- This is an automatically generated file. DO NOT EDIT. Edit gen_menu_docs.py instead -->
+
+# IronOS Settings Menu
+
+The below breaks down the menu's and what each setting means.
+    
+## Menu Categories
+
+In the menu there are a few main categories that are used to keep the list manageable.
+
+### Category: Power settings
+
+Menu for settings related to power. Main settings to do with the input voltage.
+
+### Category: Soldering settings
+
+Setttings for soldering mode, such as tip temperature, boost temps, sleep temps.
+
+### Category: Sleep mode
+
+Settings to do with power saving, such as sleep mode and shutdown modes.
+
+### Category: User interface
+
+User interface related settings, such as Units.
+
+### Category: Advanced settings
+
+Advanced settings. Misc catchall for settings that dont fit anywhere else or settings that require some thought before use.
+
+## Settings
+
+These are all of the settings possible in the menu.
+**Not all settings are visible for all devices.**
+For example, the TS100 does not have USB-PD settings.
+
+When using the device, if unsure you can pause (press nothing) on a setting and after a short delay help text will scroll across the screen.
+This is the "on device help text".
+
+### Setting: Power source
+
+When device is powered by a battery, adjusts the low voltage threshold for when the unit turns off the heater to protect the battery.
```

Comment:
When the device is powered by a battery, `this` adjusts the low voltage threshold for when the unit `should` turn off the heater to protect the battery.

Reply:
Ah this might be an opinion. I was trying to write succinctly so the translation editor page isnt huge. Though agree it probably reads easier with the extra works.